3-cloud-computing/3.1-cloud.md

```diff
@@ -2,13 +2,18 @@
 
 ### Cloud Computing Platforms
 
-Cloud computing platforms offer different services such as virtual machines, data storage, networking and load balancing, static content hosting and much more. 
+Cloud computing platforms offer different services such as virtual machines, data storage, networking and load balancing, static content hosting and much more. Cloud prividers
+
```

Comment:
Small misspelling with "prividers". Also, remove the extra newlines so it reads as

"Cloud computing platforms offer different services such as virtual machines, data storage, networking and load balancing, static content hosting and much more. Cloud providers also offer varying ecosystems of IaaS, CaaS, PaaS and FaaS.

Reply:
fixed